src/css/tui-editor-contents.css

```diff
@@ -221,13 +221,22 @@
     list-style: none;
     padding-left: 22px;
     margin-left: -22px;
+}
+
+.tui-editor-contents ul li.task-list-item:before {
```

Comment:
It need to consider when `ol` have task list.
How about this code?
```css
.tui-editor-contents .task-list-item:before
```

Reply:
Will do, I just copied the selector I found in the devtool